src/app/dashboard/dashboard.component.ts

```diff
@@ -1,7 +1,7 @@
 import { Component, OnInit } from '@angular/core';
 
 import { UserService } from '../shared/user.service';
-import { CouchService } from '../shared/couchdb.service';
+import { CouchService } from '../shared/couchdb.service';achievement

```

Comment:
This addition is an error.  Please remove.

Reply:
@paulbert I have removed the required part.